README.md

```diff
@@ -57,7 +57,7 @@ is given below:
    Algorithm:     Nelder-Mead
 
 * Convergence measures
-   √(Σ(yᵢ-ȳ)²)/n ≤ 1.0e-08
+   √(Σ(yᵢ-ȳ)²)/n ≤ 1.0e-08
```

Comment:
Not clear to me what changed here?

Reply:
it's showing the output and explaining the steps.

Reply:
Nvm, I think I incorrectly used the GitHub mobile app.